index.bs

```diff
@@ -334,6 +334,29 @@ partial interface NeuralNetworkContext {
 };
 </script>
 
+### reshape ### {#api-neuralnetworkcontext-reshape}
+<script type=idl>
+partial interface NeuralNetworkContext {
+  Operand reshape(Operand input, sequence<long> newShape);
+};
+</script>
+
+<div algorithm=reshape>
+    **Arguments:**
+        - *input*: an {{Operand}}. The input N-D tensor.
+        - *newShape*: a sequence of {{long}}. The logical shape of the output
+            tensor. The number of elements implied by *newShape* must be the
+            same as the number of elements in the input tensor. At most one
+            dimension of the *newShape* can be -1. In this case, the size of
+            that dimension is computed so that the total size remains constant.
+
```

Comment:
Suggest removing the word "N-D" here as it doesn't seem to provide any more context. The input is simply "The input tensor". Also, the word "logical" here seems superfluous. 

Suggest: "Only one component of `newShape` can be the special value of -1. The size of the dimension with the value -1 is computed so that the total size remains constant."

Reply:
Thanks for the inputs. I fixed them in the new commit. Please take a look.